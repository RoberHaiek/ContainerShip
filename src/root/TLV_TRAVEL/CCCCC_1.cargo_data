#<uid>,<weight>,<dstPort>
conteiner7,1,DDDDD
conteiner8,1,DDDDD
