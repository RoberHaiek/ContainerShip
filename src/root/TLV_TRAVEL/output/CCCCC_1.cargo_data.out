container3,unload,1,1,1
container5,unload,1,1,1
container7,load,1,1,1
container8,load,1,1,1
container5,load,1,1,1
container5,load,1,1,1