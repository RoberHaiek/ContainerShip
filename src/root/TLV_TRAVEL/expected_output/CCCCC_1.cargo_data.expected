container3,unload,1,1,1
container7,load,1,1,1
container8,load,1,1,1